src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import React, { useState } from 'react';
+
 import './App.scss';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { getUserById } from './services/getUser';
+import { newTodoId } from './services/newTodoId';
+import { TodoWithUser } from './types/TodoWithUser';
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+  const [currentUser, setCurrentUser] = useState(0);
+  const [currentUserError, setCurrentUserError] = useState(false);
+
+  const [todos, setTodos] = useState(todosFromServer);
```

Comment:
would be great to give a type for this state to avoid mistakes

Reply:
correct, please write types for states